Patches/Vanilla Weapons Expanded/Ammo/ChargeTriplet.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8" ?>
+
+<Patch>
+
+  <Operation Class="PatchOperationFindMod">
+    <mods>
+      <li>Vanilla Weapons Expanded</li>
+    </mods>
+    <match Class="PatchOperationSequence">
+      <operations>
+        
+        <li Class="PatchOperationAdd">
+          <xpath>Defs</xpath>
+          <value>
+            <CombatExtended.AmmoSetDef>
+              <defName>AmmoSet_8x35mmCharged_Triple</defName>
+              <label>8x35mm Charged</label>
+              <ammoTypes>
+                <Ammo_8x35mmCharged>Bullet_8x35mmCharged_Triple</Ammo_8x35mmCharged>
+                <Ammo_8x35mmCharged_AP>Bullet_8x35mmCharged_AP_Triple</Ammo_8x35mmCharged_AP>
+                <Ammo_8x35mmCharged_Ion>Bullet_8x35mmCharged_Ion_Triple</Ammo_8x35mmCharged_Ion>
+              </ammoTypes>
+              <similarTo>AmmoSet_ChargedRifle</similarTo>
+            </CombatExtended.AmmoSetDef>
+          </value>
+        </li>
+
+        <li Class="PatchOperationAdd">
+          <xpath>Defs</xpath>
+          <value>
+            <ThingDef ParentName="Base8x35mmChargedBullet">
+              <defName>Bullet_8x35mmCharged_Triple</defName>
+              <label>8x35mm Charged bullet</label>
+              <projectile Class="CombatExtended.ProjectilePropertiesCE">
+                <damageAmountBase>19</damageAmountBase>
+                <secondaryDamage>
```

Comment:
In theory, with the correct parent, most of this stuff should be inherited, meaning you don't have to duplicate it here.

Reply:
The only difference is the pelletCount and the spreadMult which is inside the projectile class. Can the rest of the nodes inside the projectile class be inherited?